rmf_fleet_adapter/src/rmf_fleet_adapter/tasks/Patrol.cpp

```diff
@@ -58,16 +58,20 @@ void add_patrol(
         for (const auto& place_msg : one_of.value())
         {
           auto place = place_deser(place_msg);
+          errors.insert(
+            errors.end(),
+            std::make_move_iterator(place.errors.begin()),
+            std::make_move_iterator(place.errors.end()));
           if (!place.description.has_value())
           {
-            return {nullptr, std::move(place.errors)};
+            continue;
```

Comment:
Shouldn't we add some form of error log here. I'm fine with continueing but it may be worth adding some form of logging.

Reply:
that makes sense since there are behavioral changes. however it doesn't look like we have the mechanisms for `RCLCPP_INFO`, and the added errors don't get propagated to logs if there is one valid waypoint.

Reply:
You could just add the error to the `errors` list.

Reply:
yeah it's been added above it, https://github.com/open-rmf/rmf_ros2/pull/378/files#diff-b4b9c89ad857e549feaf42ebb254fa0e35e91513bdebc3cb001ec51931f25e77R61